modules/storages/app/workers/storages/health_status_mailer_job.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+#-- copyright
+# OpenProject is an open source project management software.
+# Copyright (C) 2012-2024 the OpenProject GmbH
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License version 3.
+#
+# OpenProject is a fork of ChiliProject, which is a fork of Redmine. The copyright follows:
+# Copyright (C) 2006-2013 Jean-Philippe Lang
+# Copyright (C) 2010-2013 the ChiliProject Team
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+#
+# See COPYRIGHT and LICENSE files for more details.
+#++
+
+module Storages
+  class HealthStatusMailerJob < ApplicationJob
+    def perform(storage:)
+      return unless Storages::Storage.exists?(storage.id)
```

Comment:
While testing I see the following error:

![image](https://github.com/opf/openproject/assets/11717656/3fadc6a9-ceea-4e4f-8bba-908d77e601f5)

I scheduled the job, then removed storage, then started worker. 

So to prevent error properly we should pass `storage_id` and find storage at the beginning of `perform`.
The second option would be get back to previous implementation and accept raising of `ActiveJob::DeserializationError` which then removes job from the que and no emails are sent.